modules/navier_stokes/src/kernels/NSTemperatureL2.C

```diff
@@ -59,16 +59,16 @@ NSTemperatureL2::computeQpJacobian()
 Real
 NSTemperatureL2::computeQpOffDiagJacobian(unsigned int jvar)
 {
-  if (jvar == _p_var)
+  if (jvar == _rho_var)
```

Comment:
Heheh, I thought I had fixed all these *many* years ago but one slipped through the cracks.